pytapo/__init__.py

```diff
@@ -355,6 +349,7 @@ def getRecordings(self, date):
         )["result"]["responses"][0]["result"]["playback"]["search_video_results"]
 
     def getCommonImage(self):
+        warn("Prefer to use a specific value getter", DeprecationWarning, stacklevel=2)
         return self.performRequest({"method": "get", "image": {"name": "common"}})
```

Comment:
This method forces the caller to know about the response structure, let's guide implementations towards specific getters instead.